tsuru/client/auth.go

```diff
@@ -271,6 +274,128 @@ func (c *TeamList) Run(context *cmd.Context, client *cmd.Client) error {
 	return nil
 }
 
+type TeamInfo struct{}
+
+func (c *TeamInfo) Info() *cmd.Info {
+	return &cmd.Info{
+		Name:    "team-info",
+		Usage:   "team-info <team>",
+		Desc:    `Shows information about a specific team.`,
+		MinArgs: 1,
+	}
+}
+
+func (c *TeamInfo) Run(context *cmd.Context, client *cmd.Client) error {
+
+	context.RawOutput()
+	team := context.Args[0]
+	u, err := cmd.GetURL(fmt.Sprintf("/teams/%v", team))
+	if err != nil {
+		return err
+	}
+	request, err := http.NewRequest("GET", u, nil)
+	if err != nil {
+		return err
+	}
+	resp, err := client.Do(request)
+	if err != nil {
+		return err
+	}
+	if resp.StatusCode == http.StatusOK {
```

Comment:
there's no need to check here for status code, If the API returns something unusual it'll exit as an error

Reply:
I think you could invert this `if` logic:

```
	if resp.StatusCode != http.StatusOK {
		return nil
	}
```

That would make the code a little cleaner (one less indentation level)